### DISCUSSION TITLE
articles/uk-trip

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#135] [Comments]
http://hutusi.com/articles/uk-trip

 11月-12月在英国雷丁出差三周，因为公务事情较多，没太多个人时间，中间只有两个休息日，分别去了伦敦和牛津，其他就是到返两日分别在班布里(Banbury)和雷丁小转了半天。

**Comment by hutusi** [comment]
OK, gitalk!

**Comment by hanxiatu-fc** [comment]
英国的食物太难吃！！！

**Comment by doraemon1293** [comment]
大佬 我也在英国huawei上班 再来的话有机会希望能聊聊?

**Comment by hutusi** [comment]
> @doraemon1293
> 大佬 我也在英国huawei上班 再来的话有机会希望能聊聊?

好啊，真是想再去英国呢。
